### ISSUE TITLE
Any tiled upscale workflow for comfy Using LLLite Blur model?

### ISSUE THREAD

**Issue by x-legion** [kohya-ss/ControlNet-LLLite-ComfyUI#2]

**Comment by kohya-ss** [comment]
I'm not familiar to tiled upscale (ControlNet-tile?). If you could give me an example of a workflow using tiled upscale, I might be able to do something.

**Comment by x-legion** [comment]
yes. (CN tile + tiled diffusion or ultimate upscale ext) for a1111 but  replicating that in comfy using CNLLite blur + something else to get upto 4k upscale without running OOM. Something that could use tiledksampler or ultimate upscale node with CNtLLite node.

**Comment by x-legion** [comment]
![sytanmod](https://github.com/kohya-ss/ControlNet-LLLite-ComfyUI/assets/97628039/badf995e-9ad1-400e-8257-3ca63b209dcd)
current workflow for testing comfyui upscale methods

**Comment by creeduk** [comment]
There is a tile model here but how would we use it as the lllite loader requires an input image the same size as the generated image, so it seems to error.
https://huggingface.co/bdsqlsz/qinglong_controlnet-lllite
